zzzz_modular_occulus/code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -0,0 +1,359 @@
+	glass_desc = "Sweet and sugary, but also very refreshing!"
+
+/datum/reagent/drink/pumpkinjuice
+	name = "Pumpkin Juice"
+	id = "pumpkinjuice"
+	description = "Used to mix with cocktails, milkshakes and lattes. Not the most appealing drink by itself however."
+	taste_description = "bland pumpkin"
+	taste_mult = 0.5
+	color = "#EDA529"
+	taste_tag = list(TASTE_LIGHT)
+
+	glass_icon_state = "pumpkinjuice"
+	glass_name = "pumpkin juice"
+	glass_desc = "Pretty bland pumpkin juice by itself. Shouldn't you be mixing this with something?."
+
+/datum/reagent/drink/watermelonjuice
+	name = "Watermelon Juice"
+	id = "watermelonjuice"
+	description = "Delicious juice made from watermelon."
+	taste_description = "sweet watermelon"
+	color = "#B83333"
+	taste_tag = list(TASTE_SWEET, TASTE_REFRESHING) // Make it refreshing. Cus it should be.
+
+	glass_icon_state = "watermelon"
+	glass_name = "watermelon juice" // Gave it a new, better sprite.
+	glass_desc = "Delicious juice made from watermelon. The perfect drink to match your watermelon swimshorts!"
+
+/datum/reagent/drink/junglejuice
+	name = "Jungle Juice"
+	id = "junglejuice"
+	description = "A very fruity tropical flavour with a mix of sour and sweet fruit juices, cooled to perfection in order to quench the thirst of a hot summer day!"
+	taste_description = "a strong mix of many sweet, sour and refreshing tropical fruits"
+	color = "#EA5C5C"
+	taste_tag = list(TASTE_SWEET, TASTE_STRONG, TASTE_SOUR, TASTE_REFRESHING) //
+
+	glass_icon_state = "junglejuice"
+	glass_name = "jungle juice"
+	glass_desc = "A very fruity tropical flavour with a mix of sour and sweet fruit juices, cooled to perfection in order to quench the thirst of a hot summer day!"
+
+
+
+
+///////MILKSHAKES///////
+
+/datum/reagent/drink/milkshake/blueberry
+	name = "Blueberry Milkshake"
+	description = "A sweet blueberry flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	taste_description = "thick creamy blueberry"
+	id = "blueberry_milkshake"
+	color = "#969CE5"
+	adj_temp = -9
+	taste_tag = list(TASTE_LIGHT, TASTE_SWEET, TASTE_SLIMEY)
+
+	glass_icon_state = "shake_blueberry"
+	glass_name = "Blueberry Milkshake"
+	glass_desc = "A sweet blueberry flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
+	glass_center_of_mass = list("x"=16, "y"=7)
+
+/datum/reagent/drink/milkshake/blueberry/New()
+	var/toppings = pick("_solo", "_straw", "_umbr", "_both")
+	glass_icon_state += toppings
+
+/datum/reagent/drink/milkshake/pineapple // We have no pineapple fruits.
+	name = "Pineapple Milkshake"
+	description = "A sweet and strong pineapple flavour with a smooth blend of cream and milk, shaked and cooled to perfection in order to guarantee palate satisfaction, all topped with a light layer of froth."
```

Comment:
Uhhh, should use dreamcrasher's find and replace function to replace all instances of "shaked" with "shaken".